src/06-loops.js

```diff
@@ -97,7 +134,14 @@ function containsForLoop(array, target) {}
  *  targetCount([ 10, 20, 10, 20, 30 ], 10);
  *  //> 2
  */
-function targetCount(array, target) {}
+function targetCount(array, target) {
+ let count = 0;
+ for(number in array)
```

Comment:
Remember that since what we care about here is a value in an `Array`, a `for...of` loop would lend itself better here, that way instead of our variable being each index, instead we would have each element.